faq/sql-faq.md

```diff
@@ -131,7 +131,9 @@ DELETE，TRUNCATE 和 DROP 都不会立即释放空间。对于 TRUNCATE 和 DRO
 
 ## 对数据做删除操作之后，空间回收比较慢，如何处理？
 
-可以设置并行 GC，加快对空间的回收速度。默认并发为 1，最大可调整为 TiKV 实例数量的 50%。可使用 `update mysql.tidb set VARIABLE_VALUE="3" where VARIABLE_NAME="tikv_gc_concurrency";` 命令来调整。
+TiDB 采用了多版本并发控制 (MVCC)，为了使并发事务能查看到早期版本的数据，删除数据不会立即回收空间，而是延迟一段时间后再进行垃圾回收 (GC)。 你可以通过修改系统变量 [`tidb_gc_life_time`](/system-variables.md#tidb_gc_life_time)的值(默认值为 `10m0s`)配置多久触发一次 GC, 即数据删除后会被保留的时限。
+
+在进行数据备份时，`tidb_gc_life_time` 的值会自动延长以顺利完成备份。

```

Comment:
Though this sentence is consistent with the English version of document, but I just notice that it's not accurate since when performing data backup, it uses "service safe point", which is known as "self-adaptive GC" to users, to protect the data from being GC-ed, rather than extending `tidb_gc_life_time`.
Can you help update it as well as the English version? Just delete this sentence, or just write "在进行数据备份、或运行长时间的事务时，自适应 GC 机制也会防止仍然需要使用的过期版本数据被过早地删除".

Reply:
Ok. I will delete "在进行数据备份时，`tidb_gc_life_time` 的值会自动延长以顺利完成备份。" because it seems useless to answer the question "对数据做删除操作之后，空间回收比较慢，如何处理？".